modules/obtaining-ocp-packages-restricted-network-cluster.adoc

```diff
@@ -0,0 +1,182 @@
+
+* Provide a source control repository. After installation, your nodes must access source code in a source code repository, such as Git.
++
+When building applications in {product-title}, your build might contain external dependencies, such as a Maven Repository or Gemfiles for Ruby applications.
+
+* Provide a registry within the disconnected environment.
+
+.Procedure
+
+On the RHEL 7 server with an internet connection, sync the repositories:
+
+. To ensure that the packages are not deleted after you sync the repository, import the GPG key:
++
+[source,terminal]
+----
+$ rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+----
+
+. Register the server with the Red Hat Customer Portal. You must use the credentials that are associated with the account that has access to {product-title} subscriptions:
++
+[source,terminal]
+----
+$ subscription-manager register
+----
+
+. Pull the latest subscription data from Red Hat Subscription Manager:
++
+[source,terminal]
+----
+$ subscription-manager refresh
+----
+
+. Attach a subscription that provides {product-title} channels.
+.. Find an available subscription pool that provides {product-title} channels:
++
+[source,terminal]
+----
+$ subscription-manager list --available --matches '*OpenShift*'
+----
+
+.. Attach a pool ID for a subscription that provides {product-title}:
++
+[source,terminal]
+----
+$ subscription-manager attach --pool=<pool_id>
+----
++
+[source,terminal]
+----
+$ subscription-manager repos --disable="*"
+----
+
+. Enable only the repositories required by {product-title} {product-version}.
++
+--
+** For cloud installations and on-premise installations on x86_64 servers, run the following command:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-ose-4.7-rpms" \
+    --enable="rhel-7-server-ansible-2.9-rpms"
+----
+--
+** For on-premise installations on IBM POWER8 servers, run the following command:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-7-for-power-le-rpms" \
+    --enable="rhel-7-for-power-le-extras-rpms" \
+    --enable="rhel-7-for-power-le-optional-rpms" \
+    --enable="rhel-7-server-ansible-2.9-for-power-le-rpms" \
+    --enable="rhel-7-server-for-power-le-rhscl-rpms" \
+    --enable="rhel-7-for-power-le-ose-4.7-rpms"
+----
+** For on-premise installations on IBM POWER9 servers, run the following command:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-7-for-power-9-rpms" \
+    --enable="rhel-7-for-power-9-extras-rpms" \
+    --enable="rhel-7-for-power-9-optional-rpms" \
+    --enable="rhel-7-server-ansible-2.9-for-power-9-rpms" \
+    --enable="rhel-7-server-for-power-9-rhscl-rpms" \
+    --enable="rhel-7-for-power-9-ose-4.7-rpms"
+----
+
+. Install required packages:
++
+[source,terminal]
+----
+$ sudo yum -y install yum-utils createrepo docker git
```

Comment:
No need to install `docker` here.

Reply:
Why remove the whole line? The following steps still need to install `yum-utils`, `createrepo`